lumigator/python/mzai/jobs/evaluator_lite/evaluator_lite/eval_metrics.py

```diff
@@ -0,0 +1,82 @@
+import evaluate
+import numpy as np
+from loguru import logger
+
+
+class EvaluationMetrics:
+    def __init__(self, metrics):
+        self._supported_metrics = {
+            "rouge": self._rouge,
+            "meteor": self._meteor,
+            "bertscore": self._bertscore,
+        }
+
+        # chosen metrics are the intersection between the provided and the supported ones
+        self._chosen_metrics = set(metrics) & set(self._supported_metrics.keys())
+        # unsupported metrics are the difference between the provided and the supporterd ones
```

Comment:
```suggestion
        # unsupported metrics are the difference between the provided and the supported ones
```

Found my own typo :D

Reply:
Addressed with https://github.com/mozilla-ai/lumigator/pull/523/commits/1d81e87220820a3f2ee886f319810f992ef8ec49